css/selectors/focus-visible-013.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<meta charset="utf-8" />
+<title>CSS Test (Selectors): Mouse focus does not match :focus-visible</title>
```

Comment:
Would also need to update this. Something like "mouse focus does not cause an outline to be shown by default".